x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/event_column_view.tsx

```diff
@@ -209,29 +109,91 @@ export const EventColumnView = React.memo<Props>(
       [eventIdToNoteIds, id, isEventPinned, onPinEvent, onUnPinEvent]
     );
 
+    const eventType = getEventType(ecsData);
+
+    const additionalActions = useMemo<JSX.Element[]>(
+      () => [
+        <InvestigateInResolverAction
+          key="investigate-in-resolver"
+          timelineId={timelineId}
+          ecsData={ecsData}
+        />,
+        ...(timelineId !== TimelineId.active && eventType === 'signal'
+          ? [
+              <InvestigateInTimelineAction
+                key="investigate-in-timeline"
+                ecsRowData={ecsData}
+                nonEcsRowData={data}
+              />,
+            ]
+          : []),
+        ...(!isEventViewer
+          ? [
+              <AddEventNoteAction
+                key="add-event-note"
+                associateNote={associateNote}
+                getNotesByIds={getNotesByIds}
+                noteIds={eventIdToNoteIds[id] || emptyNotes}
+                showNotes={showNotes}
+                toggleShowNotes={toggleShowNotes}
+                updateNote={updateNote}
+                status={status}
+                timelineType={timelineType}
+              />,
+              <PinEventAction
+                key="pin-event"
+                onPinClicked={handlePinClicked}
+                noteIds={eventIdToNoteIds[id] || emptyNotes}
+                eventIsPinned={isEventPinned}
+                timelineType={timelineType}
+              />,
+            ]
+          : []),
+        ...(eventType === 'signal'
+          ? [
+              <AlertContextMenu
+                key="alert-context-menu"
+                ecsRowData={ecsData}
+                nonEcsRowData={data}
+                timelineId={timelineId}
+              />,
+            ]
+          : []),
```

Comment:
consider returning a disabled `AlertContextMenu` here to eliminate the gap shown in the screenshot below:

<img width="874" alt="gap" src="https://user-images.githubusercontent.com/4459398/91326718-e5c65800-e781-11ea-9de0-d5201dbf02dd.png">